src/config.rs

```diff
@@ -43,11 +42,12 @@ pub fn load_from(path: impl AsRef<Path>) -> anyhow::Result<Config> {
 }
 
 pub fn installed_config() -> Option<PathBuf> {
-    dirs::home_dir().map(|home| home.join(".gurk.toml"))
+    dirs::config_dir().map(|config_dir| config_dir.join("gurk/gurk.toml"))
 }
 
 fn default_data_path() -> PathBuf {
-    dirs::home_dir()
-        .map(|home| home.join(".gurk.data.json"))
+    dirs::data_dir()
+        .map(|data_dir| data_dir.join("gurk/gurk.data.json"))
```

Comment:
I just realized that you need to ensure that `<data_dir>/gurk` folder exists in `App::save` function. It should be possible to achieve that by using `std::fs::create_all_dirs` and `std::path::Path::parent`.

Reply:
done